lib/rules/block-indentation.js

```diff
@@ -254,9 +252,9 @@ module.exports = class BlockIndentation extends Rule {
 
       this.log({
         message: warning,
+        node,
         line: node.loc.end.line,
         column: node.loc.end.column,
```

Comment:
This is specifically using `end` values from `loc`, rather than the more conventional `start`, which is why it appears inconsistent but is actually not.